docs/source/api/safe/patched_conics.rst

```diff
@@ -0,0 +1,5 @@
+Patched Conics Module
```

Comment:
Oops... patched conics should be under `poliastro.threebody`! https://github.com/poliastro/poliastro/issues/500 Again, _let's leave that for a future pull request_.